index.html

```diff
@@ -25,39 +27,39 @@
 
       <section>
```

Comment:
I think you could fix your margin with this section by adding a margin to it in the CSS :-)